cmake/app/boilerplate.cmake

```diff
@@ -89,6 +89,8 @@ if((NOT DEFINED ZEPHYR_BASE) AND (DEFINED ENV_ZEPHYR_BASE))
   set(ZEPHYR_BASE ${ENV_ZEPHYR_BASE} CACHE PATH "Zephyr base")
 endif()
 
+find_package(ZephyrExtension PATHS ${ZEPHYR_BASE}/../* QUIET NO_DEFAULT_PATH NO_POLICY_SCOPE)
```

Comment:
I would suggest `ZephyrPackageExtension` to match the name in https://github.com/zephyrproject-rtos/zephyr/tree/master/share

Reply:
you are referring to the path here, not the package name (The Zephyr package is simply named `Zephyr`, which is seen when doing: `find_package(Zephyr)`).

Reply:
Sure, but it is a package. Anyway the current name is better